spec/features/collection_drafts/forms/data_contacts_form/data_contacts_form_data_center_contacts_spec.rb

```diff
@@ -131,7 +133,7 @@
             it 'still displays the Data Center on the preview page' do
               within '.data-centers-cards' do
                 expect(page).to have_content(data_center_short_name)
-                expect(page).to have_content(data_center_long_name)
+                expect(page).to have_content(data_center_long_name_without_space)
```

Comment:
Why do you test for both here?

Reply:
see comment above about the whitespace in the KMS long name and how it's treated in an input field and otherwise.